src/content/api/cli.md

```diff
@@ -31,20 +32,39 @@ For proper usage and easy distribution of this configuration, webpack can be con
 
 Read the [installation guide](/guides/installation) if you don't already have webpack and CLI installed.
 
+## Commands
 
-## Usage with configuration file
+webpack-cli offers a variety of commands to make working with webpack easy.
+
+| Command   | Alias | Description                                            |
+| --------- | ----- | ------------------------------------------------------ |
+| `init`    | c     | Initialize a new webpack configuration                 |
+| `migrate` | m     | Migrate a configuration to a new version               |
+| `loader`  | l     | Scaffold a loader repository                           |
+| `plugin`  | p     | Scaffold a plugin repository                           |
+| `info`    | i     | Outputs information about your system and dependencies |
+| `serve`   | s     | Run the webpack Dev Server                             |
+
+## Usage
+
+### With configuration file
 
 ```bash
 webpack [--config webpack.config.js]
 ```
 
 See [configuration](/configuration) for the options in the configuration file.
 
+### Without configuration file
+
+```sh
+webpack <entry> [<entry>] -o <output-path>
+```
```

Comment:
Does this mean?

```suggestion
```sh
webpack --entry [<entry-file>] -o <output-path>
```
```

Reply:
both are valid actually, you can pass entries without entry flag